src/nvim/edit.c

```diff
@@ -3973,17 +3978,18 @@ static void ins_compl_add_dict(dict_T *dict)
 ///
 /// @param[in]  tv  Object to get matches from.
 /// @param[in]  dir  Completion direction.
+/// @param[in]  fast  use fast_breakcheck() instead of ui_breakcheck().
```

Comment:
isn't this supposed to be `os_breakcheck` instead of `ui_breakcheck`? At edit.c:2325 it's called `os_breakcheck` so I assume nvim chose a different name. I know this is just a comment but that *might* cause a confusion in the future.

Reply:
Oh, `ui_breakcheck()` is Vim8 name.  I will fix it later.